src/celeritas/track/detail/TrackSortUtils.hh

```diff
@@ -89,13 +85,14 @@ struct alive_predicate
     }
 };
 
-struct action_comparator
+template<class Id>
+struct id_comparator
```

Comment:
While you're at it, could you fix the spelling of these structs `IdComparator`, `AlivePredicate`, ...?